pypyr/log/logger.py

```diff
@@ -19,6 +22,30 @@ def set_logging_config(log_level, handlers):
         handlers=handlers)
 
 
+class NoticeLogger(logging.getLoggerClass()):
+    """Root logger class with adding extra notice logging method.
+    """
+    def notice(self, msg, *args, **kwargs):
+        """Log a message with severity 'NOTICE' on the root logger.
+        """
+        if self.isEnabledFor(NOTICE):
+            self._log(NOTICE, msg, args, **kwargs)
+
+
+def set_up_notice_log_level():
+    """Add up a global notice severity to the python logging package.
+
+    NOTICE severity is logging level between INFO and WARNING.
+    By default it outputs only echo step and step name
+    with description.
+    """
+
+    logging.addLevelName(NOTICE, "NOTICE")
+    logging.NOTICE = NOTICE
+
+    logging.setLoggerClass(NoticeLogger)
```

Comment:
I'm not sure is it ok to patch globally `logging` package by adding new `NOTICE` severity

Reply:
see code review comment